.release-it.json

```diff
@@ -7,5 +7,8 @@
 		"autoGenerate": true,
 		"release": true,
 		"releaseName": "v${version}"
+	},
+	"npm": {
+		"publishArgs": ["--include-workspace-root"]
```

Comment:
[Question] Is this necessary for this repo template? Looking at https://docs.npmjs.com/cli/v9/commands/npm-publish#include-workspace-root I don't think we use workspaces in a way this is useful for? _(I'm not familiar with this area of stuff, sorry)_

Reply:
oh yeah, good spot - that should be `--provenance`